### DISCUSSION TITLE
How to point to src folder instead of dist folder in development mode?

### DISCUSSION THREAD

**Discussion by rohith2710** [AstroDraw/AstroChart#77] [General]
Hi,  
I'm new to this repository, I have checked out this repo to understand modify some changes as per my need while rendering the svg. but I'm not sure how do I point to this path 'Astrochart/AstroChart/project/src/index.ts' instead of  <script src="../../../dist/astrochart.js"></script> this in radix.html file. 

I have made this change in project/examples/radix/radix.html, like below :
<script src="../../src/index.ts"></script>

but as the first path is in typescript , browser is unable to compile this, can anyone help me with this?

@afucher  @Kibo  @harlantwood  @ilyai  @rdwz

**Comment by afucher** [comment]
Did you tried to build the project?
`npm build`

**Reply by rohith2710** [reply]
Yes I ran it, but it generates minified version files right?, How would I debug Chart.js file in chrome developer tools?

let us say I want to debug calibrate in chart.ts while running radix.html, how do I do this?

**Comment by harlantwood** [comment]
The general concept I think you want here is a sourcemap — this will tie your dev tools (etc) js line to the original ts line.  We are using webpack, which can generate sourcemaps, but not sure if we are doing so.

**Reply by afucher** [reply]
@rohith2710 I just pushed a new branch (https://github.com/AstroDraw/AstroChart/tree/source-map-dev) that now enable you to generate the files with source-map.
Can you test it please?
`npm run dev` or `npm run dev:w` will generate the source-maps so you can debug in the developer tools.

**Reply by rohith2710** [reply]
@afucher  Perfect!, Yes I can see source-map now using npm run dev.
Thank you!
